### PR TITLE
add lua mathematical library entry

### PR DESCRIPTION
**Pull request by navea330**
### Description

This PR adds a Lua mathematical library entry. This is part of the AoT Internship Lua project.

### Type of Change

- Adding a new entry

### Checklist

- [X ] All writings are my own.
- [ X] My entry follows the Codecademy Docs style guide.
- [X ] My changes generate no new warnings.
- [ X] I have performed a self-review of my own writing and code.
- [X ] I have checked my entry and corrected any misspellings.
- [X ] I have made corresponding changes to the documentation if needed.
- [ X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/lua/concepts/mathematical-library/mathematical-library.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  - 'Numbers'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-lua'
+---
+The **Mathematical Library** holds Lua built-in functions that relate to math. Just as `print()` is a built-in function, Lua offers other functions for specific use cases.
+
+## Math Functions
+Lua’s Mathematical Library contains various math functionalities, but here are a few most commonly used functions:
+
+#### `math.max(x, ..)`
```
Comment (lucwhite) [inline content/lua/concepts/mathematical-library/mathematical-library.md:19]: These should likely be terms but thats out of scope for the internship. Make note of this in the handoff doc

File: content/lua/concepts/mathematical-library/mathematical-library.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Description: "Lua's Mathematical Library holds built-in functions relating to math." # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+ - 'Computer Science'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Math'
+  - 'Numbers'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-lua'
```
Comment (lucwhite) [inline content/lua/concepts/mathematical-library/mathematical-library.md:12]:
```suggestion
  - 'learn-lua'
  - 'paths/computer-science'
```

File: content/lua/concepts/mathematical-library/mathematical-library.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+#### `math.pow(x, y)`
+Returns x to the y power (x^y).
+
+#### `math.random(a, ...)`
+Returns a random integer between 0 and 1 when no arguments are given, a random integer from 1 to a when one argument is given, or a random integer betwen a and b when two arguments are given.
+
+#### `math.sqrt(x)`
+Returns square root of x (√x).
+
+To see a full list of math functions, visit [the Lua documentation page](https://www.lua.org/manual/5.1/manual.html#5.6).
```
Comment (lucwhite) [inline content/lua/concepts/mathematical-library/mathematical-library.md:34]:
```suggestion
To see a full list of math functions, visit the [Lua documentation page](https://www.lua.org/manual/5.1/manual.html#5.6).
```

**Review by lucwhite — CHANGES_REQUESTED** [review]
Great work! Just a couple small changes

File: content/lua/concepts/mathematical-library/mathematical-library.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+---
+Title: 'Mathematical Library' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: "Lua's Mathematical Library holds built-in functions relating to math." # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
```
Comment (navea330) [inline content/lua/concepts/mathematical-library/mathematical-library.md:3]:
```suggestion
Description: "Lua's Mathematical Library holds built-in functions relating to math" # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
```

File: content/lua/concepts/mathematical-library/mathematical-library.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Math'
+  - 'Numbers'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-lua'
+---
+The **Mathematical Library** holds Lua built-in functions that relate to math. Just as `print()` is a built-in function, Lua offers other functions for specific use cases.
+
+## Math Functions
```
Comment (navea330) [inline content/lua/concepts/mathematical-library/mathematical-library.md:16]:
```suggestion
## Math Functions

```

**Review by yangc95 — APPROVED** [review]
@navea330 Great base for building on top of! The functions you listed out will make great future terms 👍

**Pull request merged**